helm/aws-operator/templates/_helpers.tpl

```diff
@@ -0,0 +1,38 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "aws-operator.name" -}}
+{{- default .Chart.Name .Values.project.name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "aws-operator.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
```

Comment:
Why do we need to replace "+"?

Reply:
This comes from https://github.com/helm/helm/blame/ec1d1a3d3eb672232f896f9d3b3d0797e4f519e3/pkg/chartutil/create.go#L368 and in turn from https://github.com/helm/helm/commit/2e0f949d49a0fccde8463ca145346c2ee0403c65

Generally k8s has some [constraints](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/identifiers.md) when it comes to what characters are allowed in certain names so I think the default Helm templates try to be "robust" here in case those variables are used in places where this could break things.

Reply:
Can we skip the replace part? I'd rather be less robust and know we don't include weird symbols in the name.

Reply:
Do we have a linter or something that would catch this early? Otherwise the errors from bad resource names will not be immediately visible right? I.e. only in chart-operator logs?

Reply:
Is this because `+` is a valid character in semver?

Reply:
You're right, it is, so I'd say it's likely.